<?xml version="1.0"?>
<launch>
    
  <node name="minipc_sytem_monitor" pkg="puma_system_monitor" type="system_monitor_node.py" output="log" respawn="true" respawn_delay="1">
    <param name="thermal_zone" value="1"/>
    <remap from="/cpu_usage" to="/puma/minipc/cpu_usage"/>
    <remap from="/cpu_temperature" to="/puma/minipc/cpu_temperature"/>
    <remap from="/memory_usage" to="/puma/minipc/memory_usage"/>
  </node>

</launch>